admin-manual/customization/theming.rst

```diff
@@ -54,23 +60,116 @@ don't have to get your hands too dirty. It's in the details where most of the
 complexities are found. Dominion is the result of a cyclic process of testing
 and refining by a large community of users, try not to underestimate that!
 
-AtoM bundles two themes: arDominionPlugin and arArchivesCanadaPlugin. Their
-names follow the naming convention of Symfony 1.x plugins, because that is how
-themes are implemented in AtoM. You may want to read more about Symfony plugins
+AtoM bundles three themes: arDominionPlugin, arDominionB5Plugin and arArchivesCanadaPlugin.
+Their names follow the naming convention of Symfony 1.x plugins, because that is
+how themes are implemented in AtoM. You may want to read more about Symfony plugins
 later following `one of their guides <http://symfony.com/legacy/doc/gentle-introduction/1_4/en/17-Extending-Symfony#chapter_17_plug_ins>`_.
 
-arDominionPlugin is the default theme, i.e. the theme that will be used in a
+arDominionB5Plugin is the default theme, i.e. the theme that will be used in a
 fresh installation. arArchivesCanadaPlugin was developed as an extension of the
 former and the following instructions will show you how to create your custom
 theme as we did with arArchivesCanadaPlugin.
 
-Assuming that you already have AtoM installed in your development environment
-(you can use our :ref:`Vagrant box <dev-env-vagrant>`), let's start
-building the plugin structure from the command line. Our theme is going to be
-called Corcovado (arCorcovadoPlugin). We are going to track its contents with
-git and publish them in a remote repository hosted by GitHub so we can enable
-others to contribute in the development. The repository is open source so you
-can use it for your own reference, see https://github.com/artefactual-labs/atom-theme-corcovado.
+The following instructions assumes that you already have AtoM installed in your
```

Comment:
2 quick comments here - first, I think it should be "The following instructions assume" (i.e. singular assume, because the instructions are plural - he assumes; they assume).

More importantly, I am not sure if the diff is showing me this correctly, but: it seems like this sentence just cuts off? If anything, I think you can probably remove that the whole sentence starting with "The following...", as it's no longer really needed with the way that things are structured. 

If you still wanted to include some sort of reference to the dev environments, and a link to the docs for them, I would suggest adding a tip admonition right under the section title below, something like: 

```
.. TIP::

   The following instructions assume you already have a n AtoM development environment set up locally. If not, we have two development-friendly environments for AtoM - see: 

   * :ref:`dev-env-vagrant`
   * :ref:`dev-env-compose`
  
```